workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationInputs/getIdentityPermissionsByIdentity.ts

```diff
@@ -3,27 +3,37 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { IdentityPermission, IdentityType } from './identityPermission';
+import { z } from 'zod';
+import { IdentityPermission, IdentityTypeParser } from './identityPermission';
 
+// eslint-disable-next-line @rushstack/typedef-var
+export const GetIdentityPermissionsByIdentityRequestParser = z
```

Comment:
Do we need request parsers at the service level?

Reply:
Yes, it is being inferred into the type. This can be utilized to validate input if necessary.